drivers/hmis/app/models/hmis/ce/configuration.rb

```diff
@@ -27,11 +29,11 @@ def days_to_retain_orphan_candidate_pools
     def values
       @values ||= AppConfigProperty.
         where(key: PROPERTIES.map { |attr| key_for(attr) }).
-        index_by { |p| p.key.to_sym }
+        index_by(&:key)
```

Comment:
This fixes a bug with `Hmis::Ce.configuration.enabled?` returning false even when there is a value for it in the config table.